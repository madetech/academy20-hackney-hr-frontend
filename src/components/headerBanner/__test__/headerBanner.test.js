import React from 'react';
import { render, unmountComponentAtNode } from "react-dom";
import renderer from 'react-test-renderer';
import HeaderBanner from '../headerBanner';

let container = null;
beforeEach(() => {
    container = document.createElement('div');
    document.body.appendChild(container);
});

afterEach(() => {
    unmountComponentAtNode(container);
    container.remove();
    container = null
});

describe("headerBanner", () => {
    test('renders the header banner without crashing', () => {
      render(<HeaderBanner />, container);
    });
    test('is rendered consistently in UI', () => {
        const component = renderer.create(
          <HeaderBanner />,
        );
        let test = component.toJSON();
        expect(test).toMatchSnapshot();
    });
})

