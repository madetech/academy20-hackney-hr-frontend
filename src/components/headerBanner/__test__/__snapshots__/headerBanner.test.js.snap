// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`headerBanner is rendered consistently in UI 1`] = `
<header>
  <div
    className="header-inner"
  >
    <div
      className="logo"
    >
      <svg>
        hackney-logo.svg
      </svg>
    </div>
    <div
      className="header-inner__name"
    />
  </div>
  <div
    className="header-lines"
  />
</header>
`;
